composer.json

```diff
@@ -25,7 +27,10 @@
         "ext-mbstring": "*",
         "ext-curl": "*",
         "composer/ca-bundle": "~1.0",
-        "symfony/options-resolver": "~2.7|~3.0"
+        "symfony/options-resolver": "~2.7|~3.0",
+        "php-http/httplug": "~1.1",
+        "php-http/curl-client": "~1.7",
```

Comment:
Well, installing a client by default became a common practice, even if it kinda destroys one of the original purposes (being able to avoid installing multiple client implementations). But I can totally understand why a library maintainer would decide to ship an OOTB working library, instead of recommending to install another one.

Maybe we should add this as a "best practice": if you want to ship your library with a client use <our most lightweight client here>. This way lib maintainers can use the same client as a sane default, so the end user don't have to install five, only the default and the choice of the user.